pkg/e2e/openshift/hypershift/verifyInstall.go

```diff
@@ -0,0 +1,84 @@
+// Package hypershift runs test units for hypershift clusters
+package hypershift
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/aws"
+	"github.com/openshift/osde2e/pkg/common/cluster"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/util"
+)
+
+var (
+	hypershiftInstallVerify = "[Suite: hypershift]"
+	workerNodes             []string
+)
+
+func init() {
+	alert.RegisterGinkgoAlert(hypershiftInstallVerify, "SD-CICD", "Diego Santamaria", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+}
+
+// Checks the installation of the hypershift cluster
+var _ = ginkgo.Describe(hypershiftInstallVerify, func() {
+	ginkgo.Context("Verify install using oc", func() {
+		util.GinkgoIt("Worker nodes are ready", func(ctx context.Context) {
```

Comment:
This test doesn't seem to be doing much, can we combine these two into one and use `checkWorkerNodesInCluster` as a requirement for checking nodes in aws?

Right now, if this first test fails to get the nodes for whatever reason, the next test will still try to run and fail so having a dependency between the two isn't the best.

I was thinking something like this:

```
Describe("hypershift", func() {
  It("worker nodes are available in aws", func() {
    By("getting the worker nodes")
    nodes, err := checkWorkerNodesInCluster(ctx)
    // expect no err
    Expect(nodes).ToNot(BeEmpty())

    By("listing the nodes in aws")
    // checkWorkerNodesInAWS()
    
    By("asserting they are equal")
    // assert the lists match
  })
})
```

Reply:
Applied in upcoming commit. 
Using the By is much better than how I had set it up.